core/src/i18n/en.yaml

```
@@ -15,11 +15,11 @@ catalog:
   subtitle: Enrich your experience with additional services that are installed outside the cluster.
   title: Service Catalog
 cluster-addons:
-  title: Cluster Addons
+  title: ClusterAddons
```

`Cluster addons` are spelled separately, and there's no reason to capitalize `addons`, as this is not a proper name :) 
See [here](https://github.com/kubernetes-sigs/cluster-addons) or [here](https://kubernetes.io/docs/concepts/cluster-administration/addons/).

```suggestion
  title: Cluster addons
```

---

If we want to follow the [SAP Styleguide](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/47f574d73c8c4cc89368203af2fbafcb.html?q=titles), we need to capitalize titles.

---

I wasn't aware of this, sorry. Let me change this now.